web/src/i18n/locales/ja.json

```diff
@@ -48,7 +48,9 @@
         "whyQ": "なぜ重要なのでしょうか？"
     },
     "findSpeaker": {
-        "title": "スピーカーを探す"
+        "title": "スピーカーを探す",
+        "pronouns": "Pronouns: {0}",
```

Comment:
Please leave translation keys blank for ja.json for now, as we have the fallback locale of English. Thanks!